4.workload_definitions.md

```diff
@@ -25,6 +23,28 @@ Here are the attributes that provide top-level information about the workload de
 
 The metadata section describes the workload definition. See [*metadata*](2.overview_and_terminology.md#Metadata).
 
+#### `Name`
+
+The `name` of the workload definition indicates _the type of this workload_.
```

Comment:
This seems to be a major change to the spec.
I'm wondering what's the purpose to have such change?

Reply:
See: https://github.com/crossplane/oam-kubernetes-runtime/issues/195

In Vela, multiple workload types will reuse same `definitionRef`.